tests/data/controlaction/EXPECTED_CONTROLACTION.txt

```diff
@@ -2,7 +2,7 @@ mutation {
   CreateControlAction(
         name: "Verovio MusicXML Converter"
         description: "MusicXML to MEI conversion"
-        actionStatus: accepted
+        actionStatus: active
```

Comment:
try to separate logical changes into separate commits. this makes it easier to review, and find bugs if necessary

Reply:
Can you confirm that active is the same as accepted was before? And also, what should the status be while running a query and after finishing it?

Reply:
We didn't make a clear decision about this. The list of statuses are listed at the bottom of https://schema.org/ActionStatusType. We should decide and update trompa documentation where necessary